.releaserc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "branches": [
+    "main",
+    "master"
```

Comment:
@and-mora Do we have a `master` branch?

Reply:
nope, i think it's derived from a copy&paste of the `.releaserc.json` which fits both branch naming